client/src/components/CancelledEngagementReports.js

```diff
@@ -68,10 +70,26 @@ export default class CancelledReports extends Component {
     let focusDetails = this.getFocusDetails()
     return (
       <div>
+        <p className="help-text">{`Number of cancelled engagement reports released since ${this.referenceDateLongStr}, grouped by advisor organization`}</p>
+        <p className="chart-description">
+          {`Displays the number of cancelled engagement reports released since
+            ${this.referenceDateLongStr}. The reports are grouped by advisor
+            organization. In order to see the list of cancelled engagement
+            reports for an organization, click on the bar corresponding to the
+            organization.`}
+        </p>
         {chartByOrg}
+        <p className="help-text">{`Number of cancelled engagement reports since ${this.referenceDateLongStr}, grouped by reason of cancelling`}</p>
```

Comment:
I have a feeling that "reason for cancellation" would be better English. However, the field label used in the Report form is "Cancelled Reason", but using that would make the sentence confusing. Perhaps ask @dspCarlson now that he's actively involved?

Reply:
Agreed. Can we have Reason for cancellation in both the text and the label?

Reply:
Sure. It would mean editing the label in `src/pages/reports/Minimal.js` and `src/pages/reports/Show.js`, and adding an explicit label to the Form.Field in `src/pages/reports/Form.js`.

Reply:
Ah, I should pay more attention. Didn't realize that it is also part of the report itself. 
While a good thing, now may not be the right time for that. I'll make a general issue to review vocabulary and add this to it. I suggest to proceed with PR as is

Reply:
@gjvoosten "Reason for Cancellation" is correct English and it sounds better.  But I understand it can't be implemented.